modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -219,42 +248,43 @@ UnitTest.asynctest('browser.tinymce.core.EditorUploadTest', (success, failure) =
     };
 
     const assertResultReusesFilename = (editor: Editor, _uploadedBlobInfo: BlobInfo, result: UploadResult[]) => {
-      Assert.eq('uploadImages reuse filename', 1, result.length);
-      Assert.eq('uploadImages reuse filename', true, result[0].status);
-      Assert.eq('uploadImages reuse filename', editor.getContent(), '<p><img src="custom.png?size=small" /></p>');
+      assert.lengthOf(result, 1, 'uploadImages reuse filename');
+      assert.isTrue(result[0].status, 'uploadImages reuse filename');
+      assert.equal(editor.getContent(), '<p><img src="custom.png?size=small" /></p>', 'uploadImages reuse filename');
 
       return result;
     };
 
     assertEventsLength(0);
-    editor.uploadImages((result) => {
+    return editor.uploadImages((result) => {
       assertResultReusesFilename(editor, uploadedBlobInfo, result);
 
       editor.uploadImages((_result) => {
         const img = editor.$<HTMLImageElement>('img')[0];
         assertEventsLength(1);
-        Assert.eq('uploadImages reuse filename', false, hasBlobAsSource(img));
-        Assert.eq('Check the cache invalidation string was added', true, img.src.indexOf('custom.png?size=small&') !== -1);
-        Assert.eq('uploadImages reuse filename', '<p><img src="custom.png?size=small" /></p>', editor.getContent());
+        assert.isFalse(hasBlobAsSource(img), 'uploadImages reuse filename');
+        assert.include(img.src, 'custom.png?size=small&', 'Check the cache invalidation string was added');
+        assert.equal(editor.getContent(), '<p><img src="custom.png?size=small" /></p>', 'uploadImages reuse filename');
         delete editor.settings.images_reuse_filename;
-      }).then(done, die);
+      });
     });
-  }));
+  });
 
-  const sUploadConcurrentImages = (editor: Editor) => Log.step('TBA', 'uploadConcurrentImages', Step.async((done, die) => {
+  it('TBA: uploadConcurrentImages', () => {
+    const editor = hook.editor();
     let uploadCount = 0, callCount = 0;
 
     const uploadDone = (result) => {
       callCount++;
       assertEventsLength(1);
 
       if (callCount === 2) {
-        Assert.eq('Should only be one upload.', 1, uploadCount);
+        assert.equal(uploadCount, 1, 'Should only be one upload.');
       }
 
-      Assert.eq('uploadConcurrentImages', '<p><img src="myimage.png" /></p>', editor.getContent());
+      assert.equal('<p><img src="myimage.png" /></p>', editor.getContent(), 'uploadConcurrentImages');
```

Comment:
Flipped `expected` and `actual` but also maybe TinyAssertions

Reply:
Fixed (not TinyAssertions though)